src/System.Runtime.Serialization.Xml/tests/CanonicalizationTestHelper.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Text;
+using System.Xml;
+
+namespace System.Runtime.Serialization.Xml.Canonicalization.Tests
+{
+    public class ReaderWriterFactory
+    {
+        public enum ReaderWriterType
+        {
+            Binary,
+            Text,
+            MTOM,
```

Comment:
MtomReader/Writer are not supported by NetCore (PNSE). Can you please remove this and other code for Mtom? Thanks.